purchase_request/models/purchase_request.py

```diff
@@ -291,6 +339,75 @@ class PurchaseRequestLine(models.Model):
                                     'Downstream Moves')
 
     orderpoint_id = fields.Many2one('stock.warehouse.orderpoint', 'Orderpoint')
+    purchase_request_allocation_ids = fields.One2many(
+        comodel_name='purchase.request.allocation',
+        inverse_name='purchase_request_line_id',
+        string='Purchase Request Allocation')
+
+    is_done = fields.Boolean('Purchase Request Line is satisfied',
```

Comment:
where is it used this field? 🤔

Reply:
It is useful to set purchase requests as done when it is fully satisfied. But I moved that to another module. I will remove the field

Reply:
That commit is not here because I started the work form the other PR. Thanks for letting me know. I will include it.